src/docker-in-docker/install.sh

```diff
@@ -347,7 +347,7 @@ fi
 
 tee /usr/local/share/docker-init.sh > /dev/null \
 << EOF
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
This might be a breaking change? I'm pretty sure there are some other places where bash is required/assumed, but just calling this out in case folks were running on an image without bash installed previously.

Reply:
I agree, reverted that change!